ui/src/views/Profile/index.jsx

```diff
@@ -14,7 +14,11 @@ import profileQuery from './profile.graphql';
 
 @hot(module)
 @withAuth
-@graphql(profileQuery)
+@graphql(profileQuery, {
+  options: () => ({
+    fetchPolicy: 'network-only',
+  }),
+})
```

Comment:
Is this a necessary practice, then -- views showing secret or user-linked data must have this fetchPolicy?  Is it worth writing that down somewhere?

Reply:
Yes, It probably makes sense to write to document this. I'll push a commit.